.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
       os: linux
 
 before_install:
-  - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
+  - echo '{"ipv6":true, "fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
```

Comment:
I meant `'{"ipv6": true, "fixed-cidr-v6": "2001:db8:1::/64"}'`.

Reply:
@Sija see https://github.com/crystal-lang/crystal/pull/5429#discussion_r158326401

Reply:
Roger that!